src/booleans.js

```diff
@@ -1,62 +1,122 @@
 function negate(a) {
-  // your code here
-};
+  return !a;
+}
 
 function both(a, b) {
-  // your code here
-};
+  //   if (a === true && b === true) {
+  //     return true;
+  //   }
+  //   return false;
+  // }
+  return a === true && b === true;
+}
 
 function either(a, b) {
-  // your code here
-};
+  if (a === true || b === true) {
+    return true;
+  } else {
+    return false;
+  }
+}
 
 function none(a, b) {
-  // your code here
-};
+  if (a === false && b === false) {
+    return true;
+  } else {
+    return false;
+  }
+}
 
 function one(a, b) {
-  // your code here
-};
+  if (a === true && b === false) {
+    return true;
+  } else if (a === false && b === true) {
+    return true;
+  } else {
+    return false;
+  }
+}
 
 function truthiness(a) {
-  // your code here
-};
+  return Boolean(a);
+}
 
 function isEqual(a, b) {
-  // your code here
-};
+  if (a === b) {
+    return true;
+  } else {
+    return false;
+  }
+}
 
 function isGreaterThan(a, b) {
-  // your code here
-};
+  if (a > b) {
+    return true;
+  } else {
+    return false;
+  }
+}
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
-};
+  if (a <= b) {
+    return true;
+  } else {
+    return false;
+  }
+}
 
 function isOdd(a) {
-  // your code here
-};
+  let remainder = a % 2;
+  if (remainder === 0) {
+    return false;
+  } else {
+    return true;
+  }
```

Comment:
The logic here is sound, but a bit long winded. All the necessary parts are there, but it can be simplified a bit!
```suggestion
  return a % 2 !== 0;
```